### PR TITLE
Pin requirements

### PR DESCRIPTION
**Pull request by imLew**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,53 +3,39 @@
...
 If you also issue `conda install python.app` in your new environment then you can have a well-behaved Mac GUI with the following command from the parent directory of ASCAM:
-`pythonw /ASCAM/src/ascam.py` 
+`pythonw /ASCAM/src/ascam.py`
 
-required packages can be found in 'requirements.txt' and installed with 
-`pip install -r requirements.txt`.
+20-03-01: Note, with the migration to Qt, some problems may be encountered on the Mac if you already have installations of Qt4+. A fresh environment (e.g. can help.
+21-05-25: Update to Big Sur - Pyqtgraph and PyQt need Python 3.8, PySide2 5.15 and the command export QT_MAC_WANTS_LAYER=1 must be issued in the Terminal.
 
-Note: Under python 3.9, axographio needs Numpy for setup, you can get around this by issuing the command `pip install numpy` before the general install command.
```
Comment (eckuru) [inline README.md:37]: I did have to do this when testing.

Comment (imLew) [reply README.md:37]: Even with python3.10? Was this on linux or mac?

Comment (eckuru) [reply README.md:37]: Yes, python 3.10 and linux. But I changed my os last weekend so it was completely fresh and numpy wasn't installed anywhere else on my system either. Maybe that's why...

Comment (imLew) [reply README.md:37]: OK, I added it back just in case.

**Review by eckuru — APPROVED** [review]

**Pull request merged**